scripts/disks/partitions.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+num=$(lsblk | wc -l)
+echo VALUE BAS number_of_devices $[$num - 1]
+for ((i=2;i<=num;i++))
+do
+	name=$(lsblk | head -n $i | tail -n 1 | tr -s '\t' ' ' | cut -f 1 -d ' ')
```

Comment:
When I run it the names start with `├─`

The output of `./partitions.sh | grep name`
```
VALUE BAS name_1 "sda"
VALUE BAS name_2 "├─sda1"
VALUE BAS name_3 "├─sda2"
VALUE BAS name_4 "└─sda5"
VALUE BAS name_5 "sr0"
```

My lsblk output
```
NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
sda      8:0    0 465.8G  0 disk 
├─sda1   8:1    0 461.9G  0 part /
├─sda2   8:2    0     1K  0 part 
└─sda5   8:5    0   3.9G  0 part [SWAP]
sr0     11:0    1  1024M  0 rom  
```